template-README.md

```diff
@@ -45,14 +53,14 @@ Example: The code in this replication package constructs the analysis file from
 ### Statement about Rights
 
 - [ ] I certify that the author(s) of the manuscript have legitimate access to and permission to use the data used in this manuscript. 
-- [ ] I certify that the author(s) of the manuscript have documented permission to redistribute/publish the data contained within this replication package. Appropriate permission are documented in the [LICENSE.txt](LICENSE.txt) file.
+- [ ] I certify that the author(s) of the manuscript have documented permission to redistribute/publish the data contained within this replication package. Appropriate permission are documented in the [LICENSE.txt] file.
```

Comment:
why did you leave the `[LICENSE.txt]` here? seems you wanted to remove the link but forgot those brackets

Reply:
ok I see below you use that for users to fill out. that may be ok then.

Reply:
brackets without link destination do not become links. Style issue. I've removed it.